### DISCUSSION TITLE
Unable to Register a new User

### DISCUSSION THREAD

**Discussion by markrichey** [knowledgefutures/pubpub#3094] [Support]
Hello,

The PubPub signup is throwing an error each time I try to create my account.
Is there a known fault with the signup process?
![PubPubFailure](https://github.com/pubpub/pubpub/assets/12082867/e51573e1-5946-486c-b7d9-360eb03941cd)

Kind Regards
Mark

**Comment by eastofesten** [comment]
Hi Mark - I'm not sure what the error was in creating your account, but I can confirm that your account was created on our end. You should be able to login with your email and chosen password (or reset your password to access the account.)

**Reply by markrichey** [reply]
Hello @sheishistoric I have been able to login to the account and the website seems considerably faster than it was this morning. Thank you for having a look!
